Rstudio-Proteomics/Dockerfile

```diff
@@ -0,0 +1,70 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         Rstudio
+# Software Version: ?
+# Description:      Rstudio-Proteomics is an adaptation from the original images provided by the Rocker-org. The container provided here contains a Rstudio server configurated with several packages dedicated to Proteomcis data analysis.Rstudio-Proteomics is an adaptation from the original images provided by the Rocker-org. The container provided here contains a Rstudio server configurated with several packages dedicated to Proteomcis data analysis.
+# Website:          https://github.com/rocker-org
+# Tags:             Genomics|Proteomics|Transcriptomics|Metabolomics|General
+# Provides:         Rstudio
+# Base Image:       bioconductor/release_core
+# Build Cmd:        docker build --rm -t biodckr/rstudio .
+# Pull Cmd:         docker pull biodckr/rstudio
+# Run Cmd:          docker run --rm  -p 8787:8787 biodckr/rstudio
+# Extra:            The Rstudio will be available via browser at:
+# Extra:            http://localhost:8787
+# Extra:            user: rstudio
+# Extra:            pass: rstudio
+#################################################################
```

Comment:
should use Docker LABEL to define most of those metadata and should specify license